main.js

```diff
@@ -3,18 +3,24 @@
 const fastify = require('fastify');
 
 const path = require('node:path');
+const fs = require('node:fs');
 
-const { Logger } = require('./src/logger.js');
+const { Logger, StreamForLogger } = require('./src/logger.js');
 const http = require('./src/http.js');
 const ws = require('./src/ws.js');
-
 const { loadApplication } = require('./src/loader.js');
+
 const APPLICATION_PATH = path.join(process.cwd(), '../NodeJS-Application');
+const LOG_FOLDER_PATH = './log';
+
+const date = new Date().toISOString().substring(0, 10);
+const LOG_FILE = path.join(LOG_FOLDER_PATH, `${date}.log`);
+const logFileStream = fs.createWriteStream(LOG_FILE, { flags: 'a' });
+const streamForLogger = new StreamForLogger(logFileStream, date);
```

Comment:
This has nothing to do with the entry point, extract it to the log file.

Moreover, the constructor itself should be responsible for opening the file.

Reply:
@nechaido Done, please review.